web/models/meeting.ex

```diff
@@ -1,19 +1,54 @@
 defmodule Meetings.Meeting do
-  defstruct title: "", date: DateTime.utc_now(), duration: 0, location: "", description: "", email: "", agenda: ""
+  defstruct title: "", location: "", date: DateTime.utc_now(), duration: 0
 
+  alias Meetings.MeetingType
+  alias Meetings.MeetingDate
```

Comment:
Gotcha, makes sense. Maybe at some point, it might make sense to consolidate database tables. Seems it works well for now.